public/pages/Alerts/containers/Alerts/Alerts.tsx

```diff
@@ -252,12 +290,20 @@ export class Alerts extends Component<AlertsProps, AlertsState> {
     this.setState({ flyoutData: alertItem ? { alertItem } : undefined });
   }
 
+  // Function to show the tooltip
+  showTooltip = (errorMessage) => {
+    this.setState({ isTooltipVisible: true, errorMessage });
+  };
+
+  // Function to hide the tooltip
+  hideTooltip = () => {
+    this.setState({ isTooltipVisible: false });
+  };
```

Comment:
1. Please make sure that we are not exposing any sensitive information with this change. Its a good practice to sanitize the error messages before exposing them to the customers.
2. One recommendation would be to expose only customer-actionable errors, like 4xx for example. Providing internal errors might confuse the users.